settings.py

```diff
@@ -69,8 +69,9 @@
 
 CSP_INCLUDE_NONCE_IN = ['script-src', 'style-src-elem']
 CSP_FONT_SRC = ('https://fonts.gstatic.com', 'data:')
-CSP_CONNECT_SRC = ("'self'", 'https://gtexportal.org', 'https://storage.googleapis.com') # used by IGV
-CSP_SCRIPT_SRC = ("'self'", "'unsafe-eval'")
+CSP_CONNECT_SRC = ("'self'", 'https://gtexportal.org', 'https://www.google-analytics.com', 'https://storage.googleapis.com') # used by IGV
```

Comment:
You may want to update the comment at the end of the line. It is not for IGV only now.